packages/contracts/contracts/foundry/Sphinx.sol

```diff
@@ -1,13 +1,19 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import { VmSafe, Vm } from "sphinx-forge-std/Vm.sol";
-
-import {
-    MerkleRootStatus,
-    SphinxLeafWithProof
-} from "@sphinx-labs/contracts/contracts/core/SphinxDataTypes.sol";
-import { ISphinxModule } from "@sphinx-labs/contracts/contracts/core/interfaces/ISphinxModule.sol";
+// We chose not to use any remappings when importing the standard forge library. This is because when our library is installed in
+// the users project we will be subject to their configured remappings. Bugs can also occur if we rely on the users installation of
+// forge-std which may not be the same exact version our library expects. To resolve both of these issues, we install the version of
+// forge-std we need ourself. We then reference it using a relative import instead of a remapping because that prevents the user from
+// having to define a separate remapping just for our installation of forge-std.
+//
+// Technically, we could use a forge-std remapping in our non-production contracts (scripts, tests, etc). We chose not too because allowing
+// that would increase the chance that we accidentally ship something that may not compile in the users project because we accidentially
+// relied on a remapping in the wrong file.
```

Comment:
It's not clear what you mean here. Does the user compile our non-production contracts (scripts, test, etc)? Regardless, this doesn't seem relevant to `Sphinx.sol`

Reply:
The user doesn't compile the non-production contracts which is why it technically is possible to use remappings in them. I've moved this comment to `Sphinx.t.sol` where it is more relevant.